static/src/js/util/url/collectionsEncoders.js

```diff
@@ -360,6 +389,8 @@ export const encodeCollections = (props) => {
     // Encode selected variables
     pg.uv = encodeSelectedVariables(projectCollection)
 
+    // Encode swodlr form variables
+    pg.swod = encodeSwodlrDownload(projectCollection)
```

Comment:
We have a wiki page where we explain/document our URL parameters, we need to make sure this is updated on that wiki page

Reply:
Will update https://wiki.earthdata.nasa.gov/display/EDSC/Earthdata+Search+URL+Parameters on merge